src/strategies/genomesdao/index.ts

```diff
@@ -33,6 +35,16 @@ export async function strategy(
   );
   const veresult: Record<string, BigNumberish> = await multi.execute();
 
+  addresses.forEach((address: any) =>
+    multi.call(address, votingAddress, 'lockEntries', [address])
+  );
+  const lockresult: Record<string, BigNumberish> = await multi.execute();
+
+  addresses.forEach((address: any) =>
+    multi.call(address, votingAddress, 'balanceOf', [address])
+  );
+  const lockedamountresult: Record<string, BigNumberish> = await multi.execute();
```

Comment:
Hi @Charles-repo There is a limit on the number of multi calls/requests a strategy can send, you can call up to 5, this change will make it 7

https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

Reply:
Hi @ChaituVR , thanks for letting me know.
How could I solve this?
We have a bit many factors to calculate voting power.

Reply:
@Charles-repo I see two multicalls are for `balanceOf` , i would suggest you remove those two, and In your space you can use `erc20-balance-of` strategy twice with this strategy,

Reply:
@ChaituVR Thanks for your guidance.

Reply:
@ChaituVR I've updated. Could you please check?